src/comms/warehouse/user.event.ts

```diff
@@ -0,0 +1,10 @@
+import type { UserStatus } from "lib/user";
+
+/**
+ * User information payload.
+ */
+export interface UserInfo {
```

Comment:
@mxcop Is this supposed to reflect the backends UserInfo data structure? if not i think it should either be changed to do that so we have some sort of parity or named differently

Reply:
I changed the name of the interface, it's not meant to reflect the UserInfo in the backend.
This was purely coincidental.